actions/delegator/setup-token/src/main.ts

```diff
@@ -103,17 +103,11 @@ async function run(): Promise<void> {
     core.info(`unsignedB64Token: ${unsignedB64Token}`);
 
     // Sign and prepare the base64 bundle.
-    const eventName = process.env.GITHUB_EVENT_NAME || "";
```

Comment:
so one problem could be that some users (TRW authors) may want to have some tests on PRs, even if the attestation is not signed. Can we handle this use case?

Reply:
I think `verify-token` would fail in that case wouldn't it?
https://github.com/slsa-framework/slsa-github-generator/blob/970b0e13262f8912dfe7522b0dbf47b7ea8a7d96/.github/workflows/delegator_generic_slsa3.yml#L92-L98

IIUC, I think this was only there to allow a presubmit on `setup-token` without doing verification.